src/panels/lovelace/cards/hui-weather-forecast-card.ts

```diff
@@ -222,9 +223,18 @@ class HuiWeatherForecastCard extends LitElement implements LovelaceCard {
               <div class="attribute">
                 ${this._config.secondary_info_attribute !== undefined
                   ? html`
-                      ${this.hass!.localize(
-                        `ui.card.weather.attributes.${this._config.secondary_info_attribute}`
-                      )}
+                      ${weatherAttrIcons[this._config.secondary_info_attribute]
+                        ? html`
+                            <ha-svg-icon
+                              class="attr-icon"
+                              .path=${weatherAttrIcons[
+                                this._config.secondary_info_attribute
+                              ]}
+                            ></ha-svg-icon>
+                          `
+                        : this.hass!.localize(
+                            `ui.card.weather.attributes.${this._config.secondary_info_attribute}`
```

Comment:
Do we still use the other translations elsewhere?

Reply:
Yes the More info